src/components/Pages/index.js

```diff
@@ -110,6 +110,7 @@ const VmDetailPageConnected = connect(
   }),
   (dispatch) => ({
     getVms: ({ vmId }) => dispatch(selectVmDetail({ vmId })),
+    checkVMAviability: ({ vmId }) => dispatch(checkVMAviability({ vmId })),
```

Comment:
I do not like this collision of names, it might be source of dummy bugs in the future or decreases readability at least.